deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/kustomization.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/assigner
+  - ingress.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+
+secretGenerator:
+  - name: identity
+    behavior: replace
```

`replace` means it must exist in `base` to be replaced here.

Since we are creating the `identity` resource in overlay only, it should be `create`. Otherwise, `kustomize build .` will fail.
```suggestion
    behavior: create
```

---

Done